### DISCUSSION TITLE
M2 iPad Pro 報到，打造 Apple Pencil 懸浮體驗

### DISCUSSION THREAD

**Discussion by dinosaur1009** [1101-GS4719-C/Python-Community-GS4719-C#801] [科技新知]
M2 晶片配備比 M1 最快可達 15% 的 8 核心 CPU，效能和效率核心都更加升級，並擁有 10 核心 GPU，為要求最嚴苛的使用者提供速度最高可快達 35% 的圖形處理效能。CPU 和 GPU 結合 16 核心神經網路引擎，每秒可進行 15.8 兆次處理，較 M1 多了 40%，讓 iPad Pro 在處理機器學習任務時更顯強大。M2 晶片也具備每秒 100GB 的統一記憶體頻寬，較 M1 多上 50%，並支援高達 16GB 的快速統一記憶體，讓多工處理和處理大型任務更加流暢。

Apple Pencil (第二代) 的懸浮功能由全新 iPad Pro 和 iPadOS 16 驅動，讓使用者能在全新維度與螢幕互動。現在，Apple Pencil 最多可在距顯示器上方 12 公釐時被偵測到，方便使用者在標示記號前先行預覽。這也讓使用者能夠更精準地進行素描或繪畫，並更輕鬆地使用 Apple Pencil 做任何事。

舉例來說，「隨手寫」功能，文字輸入框會自動在 Apple Pencil 靠近螢幕時變大，手寫字樣也能更快轉換為文字。第三方 App 也可以利用此項新功能，提供嶄新的標記與繪畫體驗。

有點好奇apple pencil會不會因此變得過度敏感（？

**Comment by Pietra1226** [comment]
不得不說蘋果的晶片真的強，即使他們慢慢擠牙膏安卓也不知道多久才會趕上(純論處理器)

**Comment by ching020721** [comment]
每個新功能出來的時候肯定都不夠成熟，但持續更進都會變得更好，除此之外，蘋果的晶片是真的很厲害，圖形處理效能對玩遊戲或進行機器學習都是很重要的，吸引指定客群在買iPad時可能會選Pro

**Comment by evanchou1204** [comment]
感覺蘋果的優勢就是有質感和人性化，雖然不管怎樣果粉還是會快樂的買下去。

**Comment by Floriaand** [comment]
可是我覺得沒有很大的創新，只是換個芯片，而且新推出的懸浮功能三星平板早就有了…

**Comment by Chiway1229** [comment]
感覺懸浮功能誤觸的問題在三星上常常聽到，不知道蘋果是否有將這部分完善好

**Comment by Doreen331** [comment]
蘋果在使用者體驗上真的做得很不錯，每次都讓我乖乖掏出我的錢，但是拜託蘋果改善一下版本限制的問題，每次下載東西就蘋果在雷😍

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
這項技術不是在三星早就出現了嗎 對於這次蘋果的推出 感覺有點敷衍

**Comment by CPCpyt** [comment]
我覺得沒有很吸引我，技術不新穎的同時消費者還要付出高昂的溢價，CP值實在有待商議。

**Comment by harrylee1971** [comment]
我覺得apple pencil能用就好了，沒有必要要用到非常高價格的筆
原子筆也不錯阿18元

**Comment by Sylvie0601** [comment]
我自己也很佩服apple pencil的技術 ! 但我覺得一支比這個價格實在太貴了，而且我也還是比較喜歡一般原子筆的手寫感，所以目前想要換不是蘋果的筆來寫 !

**Comment by Guagua1234** [comment]
現在買ipad的人通常都會加買類紙模跟筆尖套，代表喜歡手寫感的人還是占大多數吧，雖然是一門技術，但感覺必要性沒有那麼高
